src/integration/java/io/pinecone/integration/controlPlane/index/serverless/CreateDescribeListAndDeleteIndexTest.java

```diff
@@ -0,0 +1,54 @@
+package io.pinecone.integration.controlPlane.index.serverless;
+
+import io.pinecone.PineconeIndexOperationClient;
+import io.pinecone.helpers.RandomStringBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openapitools.client.model.*;
+
+import java.util.Objects;
+
+import static io.pinecone.helpers.IndexManager.isIndexReady;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class CreateDescribeListAndDeleteIndexTest {
```

Comment:
**nit:** Can we align the names of the `/pod` and `/serverless` test classes? They diverge a bit and it seems like they're going to be doing very similar things.

It's called `CreateListAndDeleteIndexTest` for `/pod/`.

Reply:
Yeah its going to be refactored into individual functions if I can.